src/app/home-page/widget/filtered-list.ts

```diff
@@ -14,17 +15,27 @@ export abstract class FilteredList extends Base implements OnInit {
   public myItems;
   public filterText;
   public isLoading = true;
+  private subject: Subject<string> = new Subject();
 
   constructor(protected userQuery: UserQuery, protected entriesService: EntriesService, protected usersService: UsersService) {
     super();
   }
 
   ngOnInit() {
     this.getMyList();
+
+    this.subject.pipe(
+      // The usual argument formDebounceTime is set to 250 which is not long enough
+      debounceTime(500),
+      distinctUntilChanged(),
+      takeUntil(this.ngUnsubscribe)
+  ).subscribe(filterText => {
```

Comment:
Indentation seems off for this line.

Reply:
Hmm...prettier should've automatically ran.  Strange it didn't.

Reply:
fixed indentation